pom.xml

```diff
@@ -137,18 +137,6 @@
                 <version>${dockstore-core.version}</version>
                 <scope>import</scope>
             </dependency>
-
-            <!--
-                  override the BOM for jackson-core version
-                  until the BOM version is upgraded
-                  several dependencies in the CLI need the newer version
-             -->
-            <dependency>
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-core</artifactId>
-                <version>2.13.1</version>
```

Comment:
Rather than downgrade, could also update to newer bom 
https://github.com/dockstore/dockstore/blob/develop/bom-internal/pom.xml#L80

I think we upgraded to satisfy some snyk checks

Reply:
OK, local test with webservice develop branch build 1.13.0-SNAPSHOT which pulls in the upgraded BOM works. 
`<dockstore-core.version>1.13.0-SNAPSHOT</dockstore-core.version>`

CLI generated pom.xml on my local machine
```
      <groupId>com.fasterxml.jackson.core</groupId>
      <artifactId>jackson-core</artifactId>
      <version>2.13.2</version>
      <scope>compile</scope>
```

However the latest webservice build on Artifactory was uploaded 2/09/2022 which is before the  jackson-core version was upgraded to 2.13.2.

![Screen Shot 2022-06-22 at 2 31 37 PM](https://user-images.githubusercontent.com/3650921/175144390-0c3087a1-da56-42d9-ab06-8baf199206d1.png)

![Screen Shot 2022-06-22 at 2 49 40 PM](https://user-images.githubusercontent.com/3650921/175144984-2a8be252-308d-4136-a8de-69c650d04573.png)

and [current CLI dockstore core is 1.13.0-alpha.6](https://github.com/dockstore/dockstore-cli/blob/9a53438aec6e21e6f322bbe407e85ef78341049f/pom.xml#L41)
![Screen Shot 2022-06-22 at 2 53 00 PM](https://user-images.githubusercontent.com/3650921/175146670-b8d1819e-13ae-47d2-861c-e69d66971cc6.png)

I'm not sure what the best solution is, maybe bump up jackson version in CLI pom?

Reply:
I would suggest making a new build, we're not really running out of alphas :wink:

Reply:
I will make a new webservice release and change the CLI pom

Reply:
Created a new webservice build which has the BOM with upgraded jackson-core version and pointed the CLI to it.